pkl/evaluator_manager_exec_windows.go

```diff
@@ -0,0 +1,25 @@
+// ===----------------------------------------------------------------------===//
+// Copyright © 2024 Apple Inc. and the Pkl project authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ===----------------------------------------------------------------------===//
+
+package pkl
+
+import (
+	"os"
+)
+
+func killProcess(proc *os.Process) error {
+	return proc.Kill()
```

Comment:
I don't think this will actually clean up child processes?

Looks like TASKKILL will do what we want:

https://stackoverflow.com/a/44551450